go/src/meguca/templates/article.go

```diff
@@ -115,14 +122,88 @@ func extractBacklinks(cap int, threads ...common.Thread) backlinks {
 	return bls
 }
 
-// Returns the stringified n + the plural or singular word from the language
-// by index word
+// Returns 3 mod-log related messages by post ID
+func parseModLog(p common.Post) (any bool, attrs string, msgs string) {
+	ln := lang.Get().Common.Posts
+	log := GetPostModLog(p.ID)
+
+	for i, val := range [3]bool{p.Banned, p.Deleted, p.MeidoVision} {
```

Comment:
I don't see how it's any more DRY to put a switch statement with 3 cases into a for loop with 3 items. All you save is the `if val {` line. Unroll this.

Reply:
Good point, I had to do something similar before, but had something after that switch statement.
There's no point in this for loop.